webapp/src/ts/modules/tasks/tasks.component.html

```diff
@@ -0,0 +1,41 @@
+<div class="filters">
+  <div class="inner">
+    <mm-navigation></mm-navigation>
+  </div>
+</div>
+
+<div class="inner">
+  <div class="inbox page">
+
+    <div id="tasks-list" class="col-sm-4 inbox-items left-pane">
+      <ul *ngIf="!error && hasTasks">
+        <mm-content-row
+          *ngFor="let task of tasksList; trackBy: listTrackBy"
+          [id]="task._id"
+          [route]="['/', 'tasks', task._id]"
+          [date]="task.date"
+          [overdue]="task.overdue"
+          [icon]="task.icon"
+          [heading]="task.contact?.name || ''"
+          [summary]="task.title"
+          [warning]="task.priorityLabel"
+          dateFormat="taskDueDate"
+          [showStatus]="false"
+          [selected]="task._id === selectedTask?._id"
+        >
+        </mm-content-row>
+      </ul>
+
+      <p *ngIf="error" class="alert alert-danger" role="alert">
+        <span translate>Error fetching tasks</span>
+        <a class="btn btn-link" (click)="refreshTaskList()" translate>Reload</a>
+      </p>
+      <p class="loading-status" *ngIf="!error && !loading && !hasTasks && !tasksDisabled" translate>No tasks found</p>
```

Comment:
We can, but I'm not sure it will be prioritized/scheduled anytime soon ;) 
It can mean breaking compatibility with older projects, we would have to create a migration to all translations ...